docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,521 @@
e81TmZm67Vwric7u0IAabVHuynBd6TqoCSypf9RghZkrmv12l0q0pi1u7VVPucpBiiuD5zIkPSMjT3fkCvN2CyunDajZohou47lrQ/JQS6Wk+IiOKDVN2tEE8V+hIVyVtfWOigfG0YAy2zwVukeYqZzdCxIzz+uGaEaE0ogqpqBl6XS1tVj44+Kk+xogOl7F2X1XwBIXjjnk948ru7EUJw3SffkFdaoemXNPh6C6tsXPnBlyy8/SoUWWbMOcN4/5FZlBVWNlndl2WJKZeNITpRu293Fhdz18/zguoVzdm1k04xsdw4dOiRD2bJEsjO1s6LqpJniaPKYKbAEs12YyI2yUCY3kev6sMgBIeMMWRQex6zs2HpUqRzz0XRN+85Um3fxaPPqdjtZkBCbRB6dCg/jideXs5rT56BLB1hfCbLMHy4Pw2vqSifV965gvc+3cIvHy3B4/SgN+qYcNFoZjx+IdEJkbBtG1JuHhHGbpQ7N1M20NpIc0RGknRkRByb0MUQIdpKq59AW7duZcyYMQwfPpzHHnsMtZmBS4gQIf5Ytm3KazEUwOXysj+7ZXXmP4puAzPoM6KrX8SqIbIi039MdzrXTTj/ZFy+KrIq5zbjqihQhYc9lT9o/xOCj5TtqBGypgUg1/5IEsIi86llV7OTvdZiNujpEBv1uw0QdUiSRLK5J2MTb2RK8j8YGHNOgAECAHUPfRNz0clNry+dLIiz2InUL22yrTXU2D+nqGgsdtvbuF3LcDp/orzsKspsNyIaCH5FdYlp4AHRCFWgDOmkKf2Xl5O9YR9FudpEzOWrd+FVVcHzV73O3s05be5npRrRrAECtK+7hkBhybAoC9Oun8xd/72KG5+5mF7DuiCZAidSaYlmkCTcKRFUTepC+Tl9qDypG670aAQQHm3FaDEiKg4F7OcJ9tqWa1NZ+iQ+KhsAnbX1BUmRkGTtBwMcPieJpUu0g0mNimjxmvSpgrRoTYfD61O56ObXObCksN4AAYhyH9+88BsvfzU/YF+LxciN109kzle3M+uzm/jumzv4x2OXYI2Pri/022+MDZtKsDT1KhJuk45wYwQ4HODxUCJMlEhh2BoM1iVJAkXh503leEePhupq3F4FgYTHojQR+RMCfEJma7G26j9++lBi06K1kAwhCFedxKg1hKtOJAQnXT6GyFql/dGn9WfohJ7a7e0T6OxevwEiIsrCVfed4W9n2qS+/sw/DQl3OzD5PKSnxgJQ3aUn1E64JCEIFy5ihINw4dImjUCUrEBmJhKg2MDo8xDmcwbUKwnQ2YBSzSiQ2CGeGU9cBBDQD1mRiVC8DBnT1f/5DnOCv62gqALF7obMTBDadSY5BFJlMKN3RYtiecFI7BCPvOQQIkGrW8gCUfe8CVeR8w6R1iGpyX5Vdicrt+WwekeuliK3URjXjsMu9qnhwY3zqmDu2wuwt8uA9HS8vkIkSWBa527yrJElEL4s1FoNHa/Xx8P/nM3Hs7/CYthGStwuPK6N/Oupd5k1a7Xm3r9nD3K3AejGKziiJKq8Mod8JnJ8Zg55TVQhocQLiIsBr5eMvFwytpRi22fG66o3gnh8CuVVYRgKfKQfzoUvvoBt27RjtVggJkb7SUnRfgczKhQWwqxZcNddcP/98K9/ab/vukv7vLDR2EBVNY2Mw4e1342ujRXfr2v+y6wle8M+ADYePMzu0tLgQr2yRL7dxuLdWlmj2cBT8+4nPk27NxSd4h8nDDt9EDe9eLl/1w82bWw25EIA725cf+T3rarCggWaZwMgrGGUeY3MsfRhdm4MuzbaOLihjJ1rKvmiqh3fxg2h0muA8NrMGwUFMH9+y/cOMG/xd1RXW4OGZ6mqzPadaeze+3PLfa1jbKDwa+TXn3P7LSOZU/Y+nx18kznlH/B/79xITFK0dp3MnAlAhL4zEYaulA4NQ0JGql1/1stWhiW9ikX/19IOC/G/Q6s9ISwWC6+99hrnnHMOV111Fd9//z333nsvSqN0XmeccUYzNYQIEeJ4oNcrDTN/BUWn+2toQgA8/NGNPHrpf9m6co9/kKH6VPoM78ID7/91xJHKXHtRaT4WWaBS5NB8RtcVH2RXRVHQciqC/JoqFh7KYmq7v1c+bo97wxFK6PC4N2C2nNW2ej27qKi4C3/OQPD/dlk24talYfT1ByCj/ABLkprJ9iFL2JAQQuBbsZL9W/O0dHe1se8NEULw9Utz+cd7N7Wpr6VeN+E+PeGyp8lYWggo8hnReUuJDr57PY3caU/pHcbs5Fic/VP8gpTEq7g7xWLcV8r5fbojyzIRWbsxRflwViq4hA5bI9d6SRF0m6Kdu1+XCkSmUTM6NEKSJYRR8PlqC9Omw5ReXXjsh1+pcXuCKIOA1WhgYg9the/D+StwZTubZDCoSx/33btrueHM8RgaxZ/r9QpxcfVpExuu5JGTwwWnn07J/pFsjV+hOYWoaL9roKO9PRaywONB6HR4cg9QJHdsMtGSJIn4ygL2PPsuvcLD8ZU6qXab0Nf4sLl0HLLHIoCUsArC9S7m7e/HyETtOAxGPS++P4NvLniQ+IN7MNVdi0KQ0K0d/YcYtElaYiKKIvPQmzP49v2lfPf+MooOlWM0G5h09hDOv2kS8clR/j5dcsYQVqzfx74DJQGrtUbhIybKQsfaiVa10BOlOugrSuhJqV9AUgKcKOxQY3BVJEF0NIq7/tzr1abPJSFDRFX9+b/gnmmkdknmi6fnsHttNrIik3nqQC67ZiQJ333kL6eUqS2unhv3liM7PBAdjSSHIVRNeFFyiSbXjaykBqQDbg09MzvTvn0ShxcexJthgg4R2qp5WQ3SikpkJKZcWq994/H6eGn2Umav2oQjzIckIMxu5LrMrlxmt/vbX1dlRvXZmm3X4/Swf9sBemdm4ttaazN2CuRKgRrd1Hil6V/Ax58u5VCPBbTrZkf4QFJApDmIHFzBgi/fYeIeO7GxYciSHq+5O2+bLGwWcbW6HCoeWcYu9P
```

Comment:
change plot title to:

Noisy regression dataset colored by label quality scores